docs/DeveloperGuide.md

```diff
@@ -362,55 +363,239 @@ testers are expected to do more *exploratory* testing.
 
 #### **Launch and Shutdown**
 
-1. Initial launch
+#### Scenario 1  
+{: .no_toc}  
+**Context:** Initial launch
 
-   1. Download the jar file and copy into an empty folder
+**Action:**
+1. Download the jar file and copy into an empty folder
+2. Double-click the jar file
```

Comment:
Since this is testing, it is probably better to also indicate that need to check that its launched in `Java 11`.

Reply:
How do we check that?